lib/cli.js

```diff
@@ -115,13 +170,32 @@ function translateOptions({
             parserOptions,
             plugins: plugin,
             rules: rule
-        },
-        overrideConfigFile: config,
-        reportUnusedDisableDirectives: reportUnusedDisableDirectives ? "error" : void 0,
-        resolvePluginsRelativeTo,
-        rulePaths: rulesdir,
-        useEslintrc: eslintrc
+        };
+    }
+
+    const options = {
+        allowInlineConfig: inlineConfig,
+        cache,
+        cacheLocation: cacheLocation || cacheFile,
+        cacheStrategy,
+        errorOnUnmatchedPattern,
+        extensions: ext,
```

Comment:
This makes the `--ext` option available in the flat config mode, but per the RFC it should not be?

https://github.com/eslint/rfcs/blob/893b49194689fcf0d34743eac75773d098264656/designs/2019-config-simplification/README.md?plain=1#L560

> This proposal removes `--ext` by allowing the same information to be passed in a config.

Reply:
Oh wow, good catch! I was so focused on making things compatible that I didn't stop to look at what I said the incompatibilities would be. I'll go through and remove this. :tada: